chapters/operatorsandexpressions.tex

```diff
@@ -1292,6 +1292,13 @@ \subsection{Constant Expressions}\doublelabel{constant-expressions}
   \lstinline!edge!, \lstinline!change!, \lstinline!sample!, and \lstinline!pre!, a function or operator with constant
   subexpressions as argument (and no parameters defined in the function)
   is a constant expression.
+\item
+  Some function calls are constant expressions regardless of the arguments:
+
+  \begin{itemize}
```

Comment:
In my experience with LaTeX, this is a paragraph break that shouldn't be there.  Hence:
```suggestion
  Some function calls are constant expressions regardless of the arguments:
  \begin{itemize}
```